pages/agent/clusters.md

```diff
@@ -0,0 +1,97 @@
+# Clusters
+
+Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+
+The following diagram shows the architectural change when enabling clusters.
+
+<%= image "clusters-architecture.png", alt: "Diagram showing existing architecture and architecture with clusters" %>
+
+Clusters encapsulate groups of agents and pipelines. This enables the following:
+
+* Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
+* Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
+* Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
+
+## Enabling clusters
+
+Any Buildkite administrator can enable clusters for an organization. Once you enable clusters, you can only disable it by contacting support.
+
+To enable clusters:
+
+1. Navigate to your [organization’s pipeline settings](https://buildkite.com/organizations/~/pipeline-settings).
+1. In _Clusters_, select _Enable Clusters_.
+
+_Clusters_ will now appear in the global navigation.
+
+## Setting up a cluster
+
+To set up a new cluster:
+
+1. Navigate to _Clusters_
+2. Click _Create a Cluster_
+3. Enter a name, description and emoji
+4. Click _Create Cluster_
+
+### Setting up queues
+
+When you create your first cluster, it will have an initial “default” queue.
+
+To create additional queues:
+
+1. Navigate to the cluster’s _Queues_
+2. Click _Create a Queue_
+3. Enter a key and description
+4. Click _Create Queue_
+
+You should [supply the queue’s key][agent-queue-key] to the agents that you wish to run build jobs on the queue.
+
+[agent-queue-key]: /docs/agent/v3/queues#setting-an-agents-queue
+
+### Using cluster agent tokens
+
+Once you’ve created your cluster and queues, navigate to the cluster’s _Agent tokens_, copy the auto-generated token and [use this token][agent-use-token] for the agents you wish to connect to the cluster.
+
+You can also create, edit, and revoke other agent tokens here.
+
+[agent-use-token]: /docs/agent/v3/tokens#using-and-storing-tokens
```

Comment:
I think this assumes you already know what you're doing. What do you think of reframing this to be about connecting an agent to a cluster? Thinking something like:

```suggestion
### Connect agents to a cluster

Agents are associated with a cluster through the cluster's agent token.

To connect an agent:

1. Navigate to the cluster's Agent tokens.
1. Copy the auto-generated token.
1. [Use the token](/docs/agent/v3/tokens#using-and-storing-tokens) with the relevant agent.

You can also create, edit, and revoke other agent tokens from the cluster's Agent tokens.
```

Reply:
Yeah, I think this is great, and I'm inclined to combine this kind of section along with the details about using the queue keys too. Give me 5 and I'll have it pushed up to #1983.